code/modules/multiz/basic.dm

```diff
@@ -0,0 +1,96 @@
+// If you add a more comprehensive system, just untick this file.
+// WARNING: Only works for up to 17 z-levels!
+var/z_levels = 0 // Each bit represents a connection between adjacent levels.  So the first bit means levels 1 and 2 are connected.
+
+// If the height is more than 1, we mark all contained levels as connected.
+/obj/effect/landmark/map_data/New()
```

Comment:
Same thing I said elsewhere about moving away from the landmark system

Reply:
Done.